packages/compass-e2e-tests/smoke-test.ts

```diff
@@ -333,6 +366,32 @@ function verifyPackagesExist(packages: Package[]): void {
   }
 }
 
+function testInstalledApp(appInfo: InstalledAppInfo) {
+  const result = spawnSync(
+    'npm',
+    [
+      'run',
+      '--unsafe-perm',
+      'test-packaged',
+      '--workspace',
+      'compass-e2e-tests',
+      '--',
+      '--test-filter=time-to-first-query',
```

Comment:
The filter will probably change once we add tests that test auto-update since it is probably easiest to test that by running the app inside an e2e test so we can use webdriverio. But this is good enough for now because it tests that the app starts up and can execute a query.